WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.h

```diff
@@ -66,10 +81,12 @@ typedef NS_ENUM(NSInteger, QuickStartTourElement) {
 @property (nonatomic, strong, nonnull) NSString *identifier;
 @property (nonatomic, strong, nullable) NSString *accessibilityIdentifier;
 @property (nonatomic, strong, nonnull) UIImage *image;
+@property (nonatomic, strong, nonnull) UIColor *imageColor;
 @property (nonatomic, strong, nullable) UIView *accessoryView;
 @property (nonatomic, strong, nullable) NSString *detail;
 @property (nonatomic) BOOL showsSelectionState;
 @property (nonatomic) BOOL forDestructiveAction;
+@property (nonatomic) BOOL hasNoAccessory;
```

Comment:
I think this might be better with a 'positive' name instead of the negative here. I personally find it potentially confusing when you end up with double negatives with properties (e.g. `hasNoAccessory = true`). Perhaps something like `showsDisclosureIndicator` could work?

Reply:
Good point that it takes extra thinking for negative naming - `showsDisclosureIndicator` sounds great, consistent with `showsSelectionState`. Will update!